### DISCUSSION TITLE
Rectangle doesn't remember window positions (on external monitors) after locking screen

### DISCUSSION THREAD

**Discussion by sjoerd-jansen** [rxhanson/Rectangle#937] [General]
- [x] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: 12.6
Rectangle version: 0.59
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): 

My windows always go back to my MacBook Pro's monitor instead of one of my external monitors when I lock and unlock the screen.

**Comment by rxhanson** [comment]
Unfortunately, this is a macOS issue. Rectangle isn't performing any actions on your windows in this scenario. I'll convert this to a discussion, since it could be treated as a feature request to handle this scenario. I don't have plans to make any adjustments to Rectangle for this, but anyone can feel free to give it a go.
